### PR TITLE
Bitcoin Core 23.2

### PR DESCRIPTION
**Pull request by fanquake**
Add release pages for Bitcoin Core v23.2.

### CODE REVIEW

**Comment by fanquake** [comment]
Pushed again to update a typo in the torrent magnet link.

**Comment by fanquake** [comment]
Pushed to fix the date.

File: _posts/en/posts/2023-05-18-release-23.2.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  Bitcoin Core 23.2 is now available.
+---
+Bitcoin Core version 23.2 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+bug fixes in this release.
+
+If you have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/23.1/
```
Comment (glozow) [inline _posts/en/posts/2023-05-18-release-23.2.md:25]: This is 23.1, should be 23.2!

Comment (fanquake) [reply _posts/en/posts/2023-05-18-release-23.2.md]: Fixed.

**Review by hebasto — APPROVED** [review]
ACK 8a11706003753d17b6f1811d85252715a31d2aa5, I've downloaded files via the torrent magnetic link and verified their hashes.

The udp://[tracker.coppersurfer.tk:6969](http://tracker.coppersurfer.tk:6969/) and udp://[tracker.leechers-paradise.org:6969](http://tracker.leechers-paradise.org:6969/) trackers were unavailable to me.

**Pull request merged**